packages/mux-uploader-react/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@mux/mux-uploader-react",
+  "version": "0.1.0-beta.1",
+  "type": "commonjs",
+  "description": "An uploader element for React that handles Mux Direct Uploads and a visual progress bar for you",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
```

Comment:
chore(blocking): Missing backwards compatible types declaration. See package.json for mux-player-react.